src/app/tests/suites/DL_LockUnlock.yaml

```diff
@@ -18,7 +18,7 @@ config:
     nodeId: 0x12344321
     cluster: "Door Lock"
     endpoint: 1
-    timeout: 25
+    timeout: 100
```

Comment:
Right, so ideally we would not need this, if we can have a shorter timer in the implementation....